en/concepts/scene/light/lightmap.md

```diff
@@ -1,74 +1,77 @@
 # Lightmapping
 
-> **Note**: currently, the lightmapping does not support Apple M1 (Silicon) architecture devices, and is expected to be supported in v3.1.
+> **Note**: the Lightmapping feature is not currently supported on Apple M1 (Silicon) architecture devices and is expected to be supported in v3.1.2.
 
-The __baking system__ is the process of finally generating light maps and applying them in the scene by calculating the influence of all light sources on the object in the engine scene. The purpose of this system is to reduce the calculation of real-time light sources, thereby improving the efficiency of the scene.
+Light maps are generated by the **Baking System** by calculating the effect of all light sources in the scene on the model. In scenes with fixed light sources, using lightmapping instead of real-time lighting calculations can reduce resource consumption and thus improve the efficiency of scene operation.
```

Comment:
```suggestion
Lightmaps are generated by the **Baking System** by calculating the effect of all light sources in the scene on the model. In scenes with fixed light sources, using lightmapping instead of real-time lighting calculations can reduce resource consumption and thus improve the efficiency of scene operation.
```